### PR TITLE
Fix javadocs in examples submodule

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

File: examples/src/main/java/io/kubernetes/client/examples/Example.java [hunk 1]
```diff
@@ -26,10 +26,10 @@
 /**
  * A simple example of how to use the Java API
  * 
- * Easiest way to run this:
+ * <p>Easiest way to run this:
```
Comment (lwander) [inline examples/src/main/java/io/kubernetes/client/examples/Example.java]: Do these tags need to be terminated?

Comment (brendandburns) [reply examples/src/main/java/io/kubernetes/client/examples/Example.java]: In theory yes, but all browsers handle unterminated <p> correctly for legacy reasons

Comment (brendandburns) [reply examples/src/main/java/io/kubernetes/client/examples/Example.java]: In theory yes, but all browsers handle unterminated <p> correctly for legacy reasons

Comment (davidxia) [reply examples/src/main/java/io/kubernetes/client/examples/Example.java]: The HTML specs say closing tags can be omitted for p tags. https://stackoverflow.com/a/33787857/553994 I got used to doing this in javadocs but can add them here if you’d like.

**Comment by brendandburns** [comment]
@lwander unless you care about the `<p>` tags, this LGTM.  I'll merge in a couple days unless I hear from you.

**Comment by lwander** [comment]
I don't mind the missing <p> tags - happy to merge now

**Comment by davidxia** [comment]
Thanks
-- 
David Xia
davidxia.com

**Pull request merged**